src/cosmicqc/analyze.py

```diff
@@ -85,8 +85,10 @@ def identify_outliers(
     for feature in feature_thresholds:
         if feature not in df.columns:
             raise ValueError(f"Feature '{feature}' does not exist in the DataFrame.")
-        outlier_df[f"Z_Score_{feature}"] = scipy_zscore(df[feature])
-        zscore_columns[feature] = f"Z_Score_{feature}"
+        outlier_df[(colname := f"{thresholds_name}.Z_Score.{feature}")] = scipy_zscore(
```

Comment:
Maybe something to address in the future when it becomes more relevant, but is it worth considering that we could in the future want to define thresholds based on other heuristics besides `Z_Score`?

Reply:
Definitely! I agree. Likely we'd need to restructure the threshold set input data structure and related details. For now I'll create an issue for this as a placeholder. If you have any opinions about next best heuristics to include please don't hesitate to add!